### PR TITLE
Feat/Update, footer, sidebar and header design 97

### LINKED ISSUE #97: Check  footer design
let's check the footer the UI design wireframes, and do exactly like graphic design do it.

image

### LINKED ISSUE #98: Check Sidebar Design
the design of the sidebar needs to be updated, let's create a new idea how we can improve this design
Let's create a section in EDEN for EOS so that there is visibility between them
image

### LINKED ISSUE #189: Implement Header
As a developer I am going to implement the new header, this for the user to have a better experience when viewing the page and find valuable information

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #97 
- Resolve #189 
- Resolve #98 

### Steps to test

1. Run Project

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Comment by leisterfrancisco** [comment]
this PR looks kind of big, please, for future PRs, try to split your code into small PRs to make it easier to review.

File: webapp/src/routes/index.js [hunk 1]
```diff
@@ -23,60 +15,60 @@ const IncomeReportDelegates = lazy(() => import('./IncomeReportDelegates'))
 const IncomeReportGeneral = lazy(() => import('./IncomeReportGeneral'))
 const Home = lazy(() => import('./Home'))
 const About = lazy(() => import('./About'))
-const Help = lazy(() => import('./Help'))
+// const Help = lazy(() => import('./Help'))
```
Comment (leisterfrancisco) [inline webapp/src/routes/index.js:18]: is it required?

File: webapp/src/routes/index.js [hunk 2]
```diff
@@ -23,60 +15,60 @@ const IncomeReportDelegates = lazy(() => import('./IncomeReportDelegates'))
...
-    path: '/help',
-    exact: true
-  },
+  // {
+  //   name: 'help',
+  //   icon: <HelpIcon />,
+  //   component: Help,
+  //   path: '/help',
+  //   exact: true
+  // },
```
Comment (leisterfrancisco) [inline webapp/src/routes/index.js:71]: do we need this?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: webapp/src/routes/index.js (hunk 2 above)
Comment (AngeloCG97) [reply webapp/src/routes/index.js:71]: Yes, I tolken with Josue about this

**Comment by leisterfrancisco** [comment]
Is this PR still in `WIP`?

**Comment by AngeloCG97** [comment]
> Is this PR still in `WIP`?

No

**Review by warnerHurtado — APPROVED** [review]
LGTM

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**